tutorials/FR/README.md

```diff
@@ -0,0 +1,93 @@
+# 🤗 Boite à Outil Éducative
+
+<aside>
+
+👋 **Bienvenue!**
+
+Nous avons assemblé une boite à outil que les professeurs du supérieur peuvent utiliser pour préparer des séances de travaux dirigés, des cours ou des devoirs. Le contenu est autonome, de manière à ce qu'il puisse être intégrer dans un cours pré-existant. Le contenu est gratuit et utilise des technologies Open Source connues (`transformers`, `gradio`, etc).
+
+Il est aussi possible de demander à un membre de l'équipe d'Hugging Face de présenter les tutoriels dans un de vos cours via l'initiative [ML demo.cratization tour](https://www.notion.so/ML-Demo-cratization-tour-with-66847a294abd4e9785e85663f5239652) !
+
+En dehors de ces tutoriels, nous mettons aussi à disposition d'autres ressources permettant d'aller plus loin dans le ML ou encore de vous assister dans la création de nouveau contenu de cours.
+
+
+</aside>
+
+## **Notre catalogue de tutoriels**
+
+### 1️⃣ Une visite à travers le Hub d'Hugging Face
+
+> Dans ce tutoriel, vous allez:
```

Comment:
```suggestion
> Dans ce tutoriel, vous allez :
```
(Same elsewhere - I think that in French there is a space before and after "signed doubles", while in English it's only after)